kuma/core/middleware.py

```diff
@@ -263,7 +263,7 @@ def process_response(self, request, response):
             # ---------
             return response
 
-        compressed_content = brotli.compress(response.content)
+        compressed_content = brotli.compress(response.content, quality=5)
```

Comment:
👍 This article, [The current state of Brotli compression](https://samsaffron.com/archive/2016/06/15/the-current-state-of-brotli-compression), suggests that level 5 is competitive with gzip in time and compression.